doc/Administration-guide/System-installation/Single-node-installation/index.md

```diff
@@ -30,14 +68,16 @@ Refer to the following guides for more details.
 Kaa requires the following third party components to be installed and configured. For more information refer to [required third party](../#third-party-components) list.
 
 <ul class="nav nav-tabs">
-  <li class="active"><a data-toggle="tab" href="#Ubuntu">Ubuntu 14.04</a></li>
+  <li class="active"><a data-toggle="tab" href="#Ubuntu16">Ubuntu 16.04</a></li>
+  <li><a data-toggle="tab" href="#Ubuntu14">Ubuntu 14.04</a></li>
```

Comment:
Instructions for Ubuntu 14.04 and 16.04 are almost the same. It would be better to make one tab for Ubuntu 14.04/16.04 and only separate 14.04 and 16.04 specific instructions using another tab inside Ubuntu 14.04/16.04 tab.